sources/packages/web/src/components/aest/students/assessment/History.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-container>
+        <body-header title="History" class="m-1"> </body-header>
+        <span>A history of assessments</span>
+        <content-group class="mt-4">
+          <DataTable
+            :value="assessmentHistory"
+            :paginator="true"
+            :rows="DEFAULT_PAGE_LIMIT"
+            :rowsPerPageOptions="PAGINATION_LIST"
+            :totalRecords="assessmentHistory.length"
+          >
+            <template #empty>
+              <p class="text-center font-weight-bold">No records found.</p>
+            </template>
+            <Column
+              field="submittedDate"
+              header="Submitted date"
+              sortable="true"
+              ><template #body="slotProps">{{
+                dateOnlyLongString(slotProps.data.submittedDate)
+              }}</template></Column
+            ><Column field="triggerType" header="Type" sortable="true"></Column
+            ><Column header="Request form"></Column
+            ><Column field="status" header="Status" sortable="true"
+              ><template #body="slotProps"
+                ><status-chip-assessment-history
+                  :status="slotProps.data.status"/></template></Column
+            ><Column
+              field="assessmentDate"
+              header="Assessment date"
+              sortable="true"
+              ><template #body="slotProps">
+                <span v-if="slotProps.data.assessmentDate">{{
+                  dateOnlyLongString(slotProps.data.assessmentDate)
+                }}</span
+                ><span v-else>-</span></template
+              ></Column
+            ><Column header="Assessment"></Column>
+          </DataTable>
+        </content-group>
+      </v-container>
+    </v-card>
+  </v-container>
+</template>
+<script lang="ts">
+import {
+  DEFAULT_PAGE_LIMIT,
+  PAGINATION_LIST,
+  AssessmentHistorySummaryDTO,
+} from "@/types";
+import { ref, onMounted } from "vue";
+import { StudentAssessmentsService } from "@/services/StudentAssessmentsService";
+import { useFormatters } from "@/composables";
+import ContentGroup from "@/components/generic/ContentGroup.vue";
```

Comment:
Not part of this PR but we can start importing components like _ContentGroup_ and _BodyHeader_ globally to avoid importing them multiple times 😉